terraform-infra-approvals/bulk-scan-shared-infrastructure.json

```diff
@@ -0,0 +1,7 @@
+{
+  "resources": [
+    {"type": "azurerm_subnet_network_security_group_association"},
+    {"type": "azure_security_group"}
```

Comment:
typo?

Reply:
I think it is `azure_security_group`, looking at the Jenkins error:

`[{managed azurerm_subnet_network_security_group_association subnet_nsg_association {azurerm } {app-gateway.tf 138}} {managed azure_security_group bulkscan {azure } {app-gateway.tf 108}}]`

Reply:
it shouldn't be that's the legacy provider which we don't use

Reply:
Ah cool....updated the TF in the bulk-scan repo and update here. Cheers 👍